CHANGELOG.md

```diff
@@ -9,17 +9,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+### Breaking changes:
+- The logic for sending Spatial position updates has been changed, the `PositionUpdateFrequency` and the `PositionDistanceThreshold` have now been replaced in the GDK settings by 4 new variables. To update your project:
+  1. Read the more detailed description of the change, you can find it by searching "Spatial Position Updates are now sent based on a different logic" inside this document.
+  2. Find any project-specific usages of the removed variables mentioned above, and replace them according to the new logic.
+  NOTE: If your project does not use custom values for the `PositionUpdateFrequency` or `PositionDistanceThreshold`, then, by default, the updates will be sent with the same frequency as before and no action is required.
+- Removed the `OnAuthorityLossImminent` Actor event.
+
 ### Features:
 - Schema bundles are now generated by default (in addition to the schema descriptor). Specifying a Schema bundle location via `-AdditionalSchemaCompilerArguments="--bundle_out=..."` will result in the bundle being written to both the default location (`build/assembly/schema/schema.sb`) _and_ the location specified by the additional arguments.
+- Spatial Position Updates are now sent based on a different logic: previously, a position update was sent to spatial if enough time has passed since the Actor's last update (computed using the SpatialUpdateFrequency) AND if the Actor has moved more than the Position Distance Threshold. This change makes the updates being sent if either of the following is true:
+  1. The time since the last sent position update is greater than or equal to PositionUpdateThresholdMinSeconds AND the distance travelled since the last update is greater than or equal to PositionUpdateThresholdMinDistance.
+  1. The time since tha last sent position update is greater than or equal to PositionUpdateThresholdMaxSeconds AND the Actor has moved any non-null amount.
```

Comment:
```suggestion
  1. The time since tha last sent position update is greater than or equal to PositionUpdateThresholdMaxSeconds AND the Actor has moved a non-zero amount.
```